### PR TITLE
Simplify code and remove some useless notes

### PR DESCRIPTION
**Pull request by tastelikefeet**

### CODE REVIEW

File: facechain/advanced-prompt/app.py
Comment (wangxingjun778) [inline facechain/advanced-prompt/app.py:1]: 两个 advanced文件夹中的app.py被删掉后， README.md 和 README_ZH.md 中的Features部分，用户示例 excution 里 cd advanced-xx, python3 app.py 也要同步修改下。 一共4处。

File: app.py [hunk 1]
```diff
@@ -91,22 +116,23 @@ def launch_pipeline(uuid,
...
-    cv2.imwrite(image_path, result)
-    # image = cv2.cvtColor(cv2.imread(image_path), cv2.COLOR_BGR2RGB)
+    if len(outputs) > 0:
+        result = concatenate_images(outputs)
+        cv2.imwrite(image_path, result)
 
-    yield ["生成完毕！", outputs_RGB]
+        yield ["生成完毕(Generating done)！", outputs_RGB]
+    else:
+        yield ["生成失败，请重试(Generating failed, please retry)！", outputs_RGB]
```
Comment (yingdachen) [inline app.py:135]: Generation failed, please try agin

File: app.py [hunk 2]
```diff
@@ -91,22 +116,23 @@ def launch_pipeline(uuid,
...
     image_path = './lora_result.png'
-    result = concatenate_images(outputs)
-    cv2.imwrite(image_path, result)
-    # image = cv2.cvtColor(cv2.imread(image_path), cv2.COLOR_BGR2RGB)
+    if len(outputs) > 0:
+        result = concatenate_images(outputs)
+        cv2.imwrite(image_path, result)
 
-    yield ["生成完毕！", outputs_RGB]
+        yield ["生成完毕(Generating done)！", outputs_RGB]
```
Comment (yingdachen) [inline app.py:133]: Your portarit photos are now ready

File: app.py [hunk 3]
```diff
@@ -122,13 +148,12 @@ def run(
         if not torch.cuda.is_available():
             raise gr.Error('CUDA is not available.')
         if instance_images is None:
-            raise gr.Error('您需要上传训练图片！')
-            # return "请传完图片后点击<开始训练>! "
+            raise gr.Error('您需要上传训练图片(Please upload photos)！')
         if len(instance_images) > 10:
-            raise gr.Error('您需要上传小于10张训练图片！')
+            raise gr.Error('您需要上传小于10张训练图片(Please upload at most 10 photos)！')
```
Comment (yingdachen) [inline app.py:153]: Please do not upload more than 10 photos

File: app.py [hunk 4]
```diff
@@ -154,7 +179,7 @@ def run(
                       output_img_dir=instance_data_dir,
                       work_dir=work_dir)
 
-        message = f'训练已经完成！请切换至 [形象体验] 标签体验模型效果'
+        message = f'训练已经完成！请切换至 [形象体验] 标签体验模型效果(Training done, please switch to the inference tab to generate photos.)'
```
Comment (yingdachen) [inline app.py:182]: Training completed. Please switch to Inference tab for photo generations.

File: app.py [hunk 5]
```diff
@@ -213,11 +232,11 @@ def train_input():
         with gr.Box():
             gr.Markdown('''
             碰到抓狂的错误或者计算资源紧张的情况下，推荐直接在[NoteBook](https://modelscope.cn/my/mynotebook/preset)上按照如下命令自行体验
-            1. git clone https://www.modelscope.cn/studios/CVstudio/cv_human_portrait.git
-            2. cd cv_human_portrait
-            3. pip install -r requirements.txt 
-            4. pip install gradio==3.35.2
-            5. python app.py
+            (If any error occurs or the waiting queue is too long, please use our [NoteBook](https://modelscope.cn/my/mynotebook/preset) with the following commands)
```
Comment (yingdachen) [inline app.py:235]: You may consider use ModelScope Notebook directly, by following the steps below:

File: app.py [hunk 6]
```diff
@@ -235,41 +254,48 @@ def inference_input():
         uuid = gr.Text(label="modelscope_uuid", visible=False)
         with gr.Row():
             with gr.Column():
-                # user_models = gr.Radio(label="风格选择", choices=['\N{fire}商务证件'], type="value", value='\N{fire}商务证件')
-                user_models = gr.Radio(label="模型选择", choices=HOT_MODELS, type="value", value=HOT_MODELS[0])
-                # flash_button = gr.Button('刷新模型列表')
+                user_models = gr.Radio(label="模型选择(Model list)", choices=HOT_MODELS, type="value",
```
Comment (yingdachen) [inline app.py:257]: Available Models

File: app.py [hunk 7]
```diff
@@ -235,41 +254,48 @@ def inference_input():
...
+                            fn=lambda model: generate_pos_prompt(model, None),
+                            label='风格模型列表(Style model list)')
 
                 with gr.Box():
                     num_images = gr.Number(
-                        label='生成图片数量', value=6, precision=1)
+                        label='生成图片数量(Number of photos)', value=6, precision=1)
                     gr.Markdown('''
-                    注意：最多支持生成6张图片!
+                    注意：最多支持生成6张图片!(Only support generating at most 6 photos one time!)
```
Comment (yingdachen) [inline app.py:276]: You may genertae at most 6 photos one time

**Pull request merged**